test/extensions/tracers/opentelemetry/http_trace_exporter_test.cc

```diff
@@ -105,7 +105,7 @@ TEST_F(OpenTelemetryHttpTraceExporterTest, CreateExporterAndExportSpan) {
   EXPECT_TRUE(trace_exporter_->log(export_trace_service_request));
 
   Http::ResponseMessagePtr msg(new Http::ResponseMessageImpl(
-      Http::ResponseHeaderMapPtr{new Http::TestResponseHeaderMapImpl{{":status", "202"}}}));
+      Http::ResponseHeaderMapPtr{new Http::TestResponseHeaderMapImpl{{":status", "200"}}}));
```

Comment:
I think this change reduce the coverage rate...I can crate a minor patch try to improve this. Hope we can land it before 1.28 cutting.

Reply:
I have no permission to push commit the source branch. 😞

Reply:
I reverted this change. I also gave you push access, can you try and see if it works?

Reply:
Let CI to try again. Hope it could pass directly.